releng/README.md

```diff
@@ -0,0 +1,55 @@
+# Release Engineering tooling
+
+This directory contains tooling to generate Prow jobs.
+
+The documentation here supersedes overlapping guidance from the
+[Release Manager handbooks][branch-manager-handbook] (which will be removed at
+a future date).
+
+- [Release branch jobs](#release-branch-jobs)
+
+## Release branch jobs
+
+This task should be done after the release is complete and previous PRs are merged. The following steps should be run from the test-infra repository's root.
+
+1. Generate the new release branch jobs:
+
+   ```shell
+   bazel run //releng:prepare_release_branch
+   ```
+
+   Alternatively, if you want to run Bazel in a container, you can use the command such as:
+
+   ```shell
+   docker run -it \
+      -v $PWD:/workspace/src/k8s.io/test-infra \
+      -v $HOME/.cache/bazel:/root/.cache/bazel \
+      -w /workspace/src/k8s.io/test-infra \
+      gcr.io/k8s-testimages/launcher.gcr.io/google/bazel:latest-test-infra run //releng:prepare_release_branch
+   ```
```

Comment:
Can we also document the `make` way of running this?

Running `make -C releng prepare-release-branch` also runs the job.

Reply:
I think we should make this the preferred way.

Reply:
+1 to using `make` as the preferred way. It's way more seamless than setting up Bazel.

Reply:
Yep, this was just copy/pasta.
Resolved in https://github.com/kubernetes/test-infra/pull/26015/commits/cd00de1bb82c6f4660bc5cc6112ba58b735d4963.